src/widgets/refinement-list/refinement-list.tsx

```diff
@@ -223,7 +226,9 @@ const renderer = ({
   searchableIsAlwaysActive,
 }: {
   containerNode: HTMLElement;
-  cssClasses: RefinementListOwnCSSClasses & { searchable: SearchBoxCSSClasses };
+  cssClasses: RefinementListOwnCSSClasses & {
+    searchable: SearchBoxRendererCSSClasses;
```

Comment:
should it not be SearchBoxCSSClasses? what's the "renderer" signifying?

Reply:
I updated this to match what was done in https://github.com/algolia/instantsearch.js/pull/4702/files#diff-f1f45efbc15d1da982a76908e834c460b724ec8753a8ba833a489ebd7645c938R158-R165

I named the `*CSSClasses` types for the renderer `*RendererCSSClasses`, they are derived from the base CSSClasses but only accept `string` and not `string[]`.